src/dbnode/persist/fs/commitlog/iterator.go

```diff
@@ -200,11 +206,24 @@ func filterCorruptFiles(corruptFiles []ErrorWithPath, predicate FileFilterPredic
 	return filtered
 }
 
+func commitLogFilesForLogging(files []persist.CommitLogFile) []string {
+	result := make([]string, 0, len(files))
+	for _, f := range files {
+		var fileSize int64
+		if fi, err := os.Stat(f.FilePath); err == nil {
+			fileSize = fi.Size()
+		}
+		result = append(result, fmt.Sprintf("path=%s, len=%d", f.FilePath, fileSize))
+	}
+	return result
+}
+
 func (i *iterator) closeAndResetReader() error {
 	if i.reader == nil {
 		return nil
 	}
 	reader := i.reader
 	i.reader = nil
+	i.log.Info("closing commit log file")
```

Comment:
Not really sure if we need this msg, I think those files are processed sequentially so seeing `opened commit log file` would be enough? Maybe just change `opened commit log file` to something like `reading commit log file`?